subsys/logging/log_output.c

```diff
@@ -177,7 +177,9 @@ static int ids_print(struct log_msg *msg,
 		total += print_formatted(log_output, "<%s> ", severity[level]);
 	}
 
-	total += print_formatted(log_output, "%s: ",
+	total += print_formatted(log_output,
+				IS_ENABLED(CONFIG_LOG_FUNCTION_NAME) ?
+				"%s." :"%s: ",
```

Comment:
Just a minor whitespace fix to make this more readable: add a space between the : and the opening " of the second string.